open-source.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: Open Source
+permalink: /open-source/
+---
+
+We believe in Open Source. We build on top of Open Source. We contribute back to the community.
+
+[Open Source Project at Maxwell](https://github.com/himaxwell?q=&type=public)
```

Comment:
@rx , as I mentioned above, until we have some useful repos in https://github.com/orgs/himaxwell/repositories?q=&type=public , let's remove this page.